crates/storage/db/src/implementation/mdbx/cursor.rs

```diff
@@ -24,13 +25,42 @@ pub type CursorRW<'tx, T> = Cursor<'tx, RW, T>;
 #[derive(Debug)]
 pub struct Cursor<'tx, K: TransactionKind, T: Table> {
     /// Inner `libmdbx` cursor.
-    pub inner: reth_libmdbx::Cursor<'tx, K>,
-    /// Table name as is inside the database.
-    pub table: &'static str,
```

Comment:
where did this go?

Reply:
it was unused and we can get the table name from `T::NAME`